### PR TITLE
test: add ThreadPoolRunStateHandler

### LINKED ISSUE #1177: [TASK] ThreadPoolRunStateHandler add unit test.
## 需求建议

为 ThreadPoolRunStateHandler 编写单元测试

### PR DESCRIPTION
**Pull request by Pan-YuJie**
Fixes #1177

Changes proposed in this pull request:
- 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: hippo4j-core/src/test/java/cn/hippo4j/core/executor/state/ThreadPoolRunStateHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+        ManyThreadPoolRunStateInfo manyThreadPoolRunStateInfo = BeanUtil.convert(poolRunStateInfo, ManyThreadPoolRunStateInfo.class);
+        manyThreadPoolRunStateInfo.setIdentify(CLIENT_IDENTIFICATION_VALUE);
+        String active = environment.getProperty("spring.profiles.active", "UNKNOWN");
+        manyThreadPoolRunStateInfo.setActive("TRUE");
+        String threadPoolState = ThreadPoolStatusHandler.getThreadPoolState(pool);
+        manyThreadPoolRunStateInfo.setState(threadPoolState);
+        Assertions.assertNotNull(manyThreadPoolRunStateInfo);
+
+
```
Comment (yanrongzhen) [inline hippo4j-core/src/test/java/cn/hippo4j/core/executor/state/ThreadPoolRunStateHandlerTest.java]: Please delete the blank line here.

File: hippo4j-core/pom.xml [hunk 2]
```diff
@@ -8,7 +8,6 @@
         <version>${revision}</version>
     </parent>
     <artifactId>hippo4j-core</artifactId>
-
```
Comment (magestacks) [inline hippo4j-core/pom.xml]: Why delete row records

**Comment by magestacks** [comment]
Please perform code formatting.

```shell
mvn spotless:apply
```

File: hippo4j-core/pom.xml (hunk 2 above)
Comment (Pan-YuJie) [reply hippo4j-core/pom.xml]: Sorry, I'll resubmit later

**Review by magestacks — APPROVED** [review]

**Pull request merged**